transports/p2p/p2putil/listeners.go

```diff
@@ -0,0 +1,96 @@
+package p2putil
+
+import (
+	"fmt"
+	"net"
+	"runtime"
+	"strings"
+
+	"github.com/bitcoin-sv/block-headers-service/config"
+	"github.com/rs/zerolog"
+)
+
+// initListeners initializes the configured net listeners and adds any bound
```

Comment:
This comment should be `InitListeners` (with Capital "I") - to match exact name of the function.

Reply:
Done